File: Domain/Models/Responses/UserSessionResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Models.Responses
{
    public class UserSessionResponse: SessionResponse
    {
        public string UserId { get; set; }
        public string UserName { get; set; }
        public double UpdateTs { get; set; }
        public string Email { get; set; }
        public string RoleId { get; set; }
        public string PhoneNumber { get; set; }
        public bool? MaritalStatus { get; set; }
        public string BirthDate { get; set; }
        public string PhotoPath { get; set; }
        public double MaxUploadFileSize { get; set; } = 0;
    }
}

File: AngularApp/src/models/responses/UserSessionResponse.ts
import { SessionResponse } from "./SessionResponse";
export interface UserSessionResponse extends SessionResponse {
    userName: string;
    userId: string;
    updateTs: number;
    email: string;
    roleId: string;
    phoneNumber: string;
    maritalStatus: boolean;
    birthDate: string;
    photoPath: string;
    maxUploadFileSize:number;
}